peribolos.yaml

```diff
@@ -1,5 +1,5 @@
 orgs:
-  uor-framework:
+  emporous:
```

Comment:
Would it be appropriate to add @d10n here?

Reply:
If we are changing roles, I would prefer a different PR. That is out of the scope of this PR.

Reply:
fair, lgtm